main/Kconfig.projbuild

```diff
@@ -148,6 +148,12 @@ menu "Project configuration"
         help
             Beacon repeat delay in seconds
 
+    config MORSE_CODE_BEACON_TEXT
```

Comment:
That looks good! I'd welcome this feature as it has been on my TODO list.

One suggestion though is to have the text to morse code conversion done on the backend (C code) instead of frontend. This way you can remove:
```
config MORSE_CODE_BEACON_TEXT
```
and only keep:
```
    config BEACON_TEXT
```
the C code should convert stored text, when needed to dots and dashes.

Front-end part could then remain unchanged.

Reply:
Thank for the suggestion! I will do as you say.

Reply:
Hello,

I got the morse enconding done on the backend so MORSE_CODE_BEACON_TEXT could be removed but I left the morse encoding in the frontend. It is possible to remove it entirely but I would like to show the user the morse string being transmitted.

Please let me know if you have any suggestion on the code. I made use of chatGPT since I am not really confortable on C.